accesspoc/dips/helpers.py

```diff
@@ -0,0 +1,7 @@
+"""Helper functions for dips app"""
+
+
+def add_if_not_empty(data_dict, key, value):
+    """Add key/value to dict if the value is not empty"""
```

Comment:
To be accurate: "if the value is truthy".
https://docs.python.org/3/library/stdtypes.html#truth-value-testing